app/helpers/frontend_asset_helper.rb

```diff
@@ -31,7 +31,7 @@ module FrontendAssetHelper
   CLI_DEFAULT_PROXY = 'http://localhost:4200'.freeze
 
   def self.assets_proxied?
-    !Rails.env.production? && cli_proxy?
+    !ENV['OPENPROJECT_SERVE_OWN_ASSETS'].present? && !Rails.env.production? && cli_proxy?
```

Comment:
Why is this new variable necessary? As far as I can tell it's only set in the docker-compose file for testing where `RAILS_ENV=test` which means `!Rails.env.production?` is false. So it's superflous, isn't it?

Reply:
Because unfortunately `cli_proxy?` always returns true, since it defaults to `CLI_DEFAULT_PROXY`. So basically `assets_proxied?` equals `!Rails.env.production?`.